build_projects/dotnet-host-build/build.sh

```diff
@@ -85,7 +85,9 @@ done < "$REPOROOT/branchinfo.txt"
 [ -z "$DOTNET_INSTALL_DIR" ] && export DOTNET_INSTALL_DIR=$REPOROOT/.dotnet_stage0/$(uname)
 [ -d "$DOTNET_INSTALL_DIR" ] || mkdir -p $DOTNET_INSTALL_DIR
 
-$REPOROOT/scripts/obtain/dotnet-install.sh --channel $CHANNEL --verbose
+
+DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/b939c88e0cf7c2e204f91a5161acdd2f5a732bf9/scripts/obtain/dotnet-install.sh"
```

Comment:
Maybe instead of a hard-coded commit, should we use the latest install script in `rel/1.0.0`?